CommonClient.py

```diff
@@ -639,6 +640,7 @@ async def shutdown(self):
             self.input_queue.put_nowait(None)
             self.input_requests -= 1
         self.keep_alive_task.cancel()
+        self.delete_old_data_package_task.cancel()
```

Comment:
Shutdown can run without ever getting to "Connected", so this needs to be put behind an if:
```suggestion
        if self.delete_old_data_package_task:
            self.delete_old_data_package_task.cancel()
```